### PR TITLE
[WOR-68] ToS passthrough

### PR DESCRIPTION
**Pull request by marctalbott**
Ticket: [WOR-68(https://broadworkbench.atlassian.net/browse/WOR-68)
Add passthrough to Sam to accept the ToS

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: automation/project/Dependencies.scala [hunk 1]
```diff
@@ -15,7 +15,7 @@ object Dependencies {
   val workbenchGoogle: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-google" % workbenchGoogleV excludeAll excludeWorkbenchModel
   val excludeWorkbenchGoogle = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-google" + scalaV)
 
-  val workbenchServiceTestV = "0.21-258d483"
+  val workbenchServiceTestV = "0.21-e8c8c3e1-SNAP"
```
Comment (marctalbott) [inline automation/project/Dependencies.scala]: will change to non-snapshot when https://github.com/broadinstitute/workbench-libs/pull/892 is merged

File: src/main/resources/swagger/api-docs.yaml [hunk 2]
```diff
@@ -7182,6 +7183,26 @@ paths:
             text/plain:
               schema:
                 type: string
+  /register/termsofservice:
```
Comment (marctalbott) [inline src/main/resources/swagger/api-docs.yaml]: I believe this needs to start with `/register` as I was having trouble with it when it didn't, but open to changing the rest of the path if there's a better one

Comment (MatthewBemis) [reply src/main/resources/swagger/api-docs.yaml]: I'm leaning towards wanting to keep the symmetry with the underlying Sam path, `/register/user/v1/termsofservice`

**Review by MatthewBemis — APPROVED** [review]
Nice! Just one comment about the path but other than that LGTM

I think we will be adding a similar GET passthrough per another AoU request, so we can treat this as the feature branch until it's all ready to go

**Review by aherbst-broad — APPROVED** [review]
Looks straightforward

**Comment by marctalbott** [comment]
This branch will not be merged until the [Sam feature branch](https://github.com/broadinstitute/sam/tree/tos-feature-branch) has been merged

**Pull request merged**